.github/workflows/ActionLint.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+
+name: "Trigger: Push action"
+permissions: {}
+
+jobs:
+  actionlint:
+    name: Actionlint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run actionlint
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
```

Comment:
It seems that we can specify the version. So how about specifying the version to avoid unexpected fail?
https://github.com/rhysd/actionlint/blob/main/docs/install.md#download-script